moped-editor/src/layouts/DashboardLayout/Footer.js

```diff
@@ -27,8 +27,10 @@ const Footer = () => {
       "https://api.github.com/repos/cityofaustin/atd-moped/tags";
     const response = await fetch(githubTagsApiEndpoint);
     const data = await response.json();
-    if (data) {
+    if (data[0]) {
```

Comment:
Not sure if `data` can ever be `null` or `undefined` here, but, if so, this could break. Looks like we've used lodash `get` elsewhere in the app so that could be a safer way.